bundle/manifests/grafana-operator.clusterserviceversion.yaml

```diff
@@ -4,13 +4,9 @@ metadata:
   annotations:
     alm-examples: '[]'
     capabilities: Basic Install
-    operators.operatorframework.io/builder: operator-sdk-v1.13.1
+    operators.operatorframework.io/builder: operator-sdk-v1.13.0+git
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
-  labels:
```

Comment:
Don't know how to run the `make bundle` command without removing the labels.
For now I can manually re-add them but I'm trying to find the way this actually should be done.
I pinged in the operator-sdk-dev slack and I hope I will get an answer soon. If anyone have a good idea how I should solve it please share.

Reply:
You need to add the labels here: https://github.com/grafana-operator/grafana-operator/blob/master/config/manifests/bases/grafana-operator.clusterserviceversion.yaml

then run `make bundle` again and they should get applied to the manifest.